ruleguard/ir/gen_filter_op.go

```diff
@@ -45,14 +46,14 @@ func main() {
 		{name: "VarConstSlice", comment: "m[$Value].ConstSlice", valueType: "string", flags: flagHasVar},
 		{name: "VarText", comment: "m[$Value].Text", valueType: "string", flags: flagHasVar},
 		{name: "VarLine", comment: "m[$Value].Line", valueType: "string", flags: flagHasVar},
-		{name: "VarGlobal", comment: "m[$Value].Global", valueType: "bool", flags: flagHasVar},
 		{name: "VarValueInt", comment: "m[$Value].Value.Int()", valueType: "string", flags: flagHasVar},
 		{name: "VarTypeSize", comment: "m[$Value].Type.Size", valueType: "string", flags: flagHasVar},
 		{name: "VarTypeHasPointers", comment: "m[$Value].Type.HasPointers()", valueType: "string", flags: flagHasVar},
 
 		{name: "VarFilter", comment: "m[$Value].Filter($Args[0])", valueType: "string", flags: flagHasVar},
 		{name: "VarNodeIs", comment: "m[$Value].Node.Is($Args[0])", valueType: "string", flags: flagHasVar},
 		{name: "VarObjectIs", comment: "m[$Value].Object.Is($Args[0])", valueType: "string", flags: flagHasVar},
+		{name: "VarObjectIsGlobal", comment: "m[$Value].Object.IsGlobal()", valueType: "bool", flags: flagHasVar},
```

Comment:
valueType is `string`, as we store a varname there. :)

Reply:
fixed it in https://github.com/quasilyte/go-ruleguard/pull/370/commits/dc16fa242b3ad35f3309b7babdcfeea7f46bf8f7 as i see